server/util/log/log.go

```diff
@@ -84,6 +97,19 @@ func LogGRPCRequest(ctx context.Context, fullMethod string, dur time.Duration, e
 	} else {
 		Infof("%s %s %s %s [%s]", "gRPC", reqID, shortPath, fmtErr(err), formatDuration(dur))
 	}
+	if logErrorStackTraces {
+		code := gstatus.Code(err)
+		if isExpectedGRPCError(code) {
+			return
+		}
+		if se, ok := err.(interface {
+			StackTrace() status.StackTrace
+		}); ok {
+			for _, f := range se.StackTrace() {
+				Infof("%+s:%d", f, f)
```

Comment:
Can you build the stacktrace into a string and make a single call to Infof? 
It'll make the stacktrace easier to read

Reply:
Done. 

they look like this:

2021/06/15 09:56:50.790 INF github.com/buildbuddy-io/buildbuddy/server/util/status.NotFoundError
	server/util/status/status.go:93
github.com/buildbuddy-io/buildbuddy/server/util/status.NotFoundErrorf
	server/util/status/status.go:99
github.com/buildbuddy-io/buildbuddy/server/remote_cache/action_cache_server.(*ActionCacheServer).GetActionResult
	server/remote_cache/action_cache_server/action_cache_server.go:150
github.com/buildbuddy-io/buildbuddy/proto/remote_execution._ActionCache_GetActionResult_Handler.func1
	bazel-out/k8-fastbuild/bin/proto/remote_execution_go_proto_/github.com/buildbuddy-io/buildbuddy/proto/remote_execution/remote_execution.pb.go:5442
github.com/buildbuddy-io/buildbuddy/server/rpc/filters.contextReplacingUnaryServerInterceptor.func1
	server/rpc/filters/filters.go:50
google.golang.org/grpc.getChainUnaryHandler.func1
	external/org_golang_google_grpc/server.go:1051
github.com/buildbuddy-io/buildbuddy/server/rpc/filters.contextReplacingUnaryServerInterceptor.func1
	server/rpc/filters/filters.go:50
google.golang.org/grpc.getChainUnaryHandler.func1
	external/org_golang_google_grpc/server.go:1051
github.com/buildbuddy-io/buildbuddy/server/rpc/filters.logRequestUnaryServerInterceptor.func1
	server/rpc/filters/filters.go:137
google.golang.org/grpc.getChainUnaryHandler.func1
	external/org_golang_google_grpc/server.go:1051